packages/web-components/src/components/va-telephone/test/va-telephone.spec.tsx

```diff
@@ -4,48 +4,63 @@ describe('formatPhoneNumber', () => {
   const contact = '8885551234';
   const N11 = '911';
   const extension = 123;
+  const vanity = 'ABCD';
   it('formats a contact number with no extension', () => {
-    expect(VaTelephone.formatPhoneNumber(contact, null)).toBe('888-555-1234');
+    expect(VaTelephone.formatPhoneNumber(contact, null, null, null)).toBe('888-555-1234');
```

Comment:
It might be nice if the `vanity` param in `formatPhoneNumber` defaulted to `null` so we didn't have to do this, but that's nitpicking.